rpc/documentation/api.md

```diff
@@ -1615,6 +1681,53 @@ wallet's relevant transactions contained therein.
 
 ___
 
+#### `TicketDetails`
+
+The `TicketDetails` message is included in responses to report transactions
+relevant to the wallet.  The message includes details such as which previous
+wallet inputs are spent by this transaction, whether each output is controlled
+by the wallet or not, the total fee (if calculable), and the earlist time the
+transaction was seen.
+
+- `bytes hash`: The hash of the given ticket.
+
+- `bytes spender_hash`: The hash of the spender (if applicable, otherwise empty).
+
+- `int64 ticket_age`: If unspent the blocks since being mined.  Otherwise,
+   the number of blocks from when it was mined to when it was spent.
+
+- `int64 ticket_price`: The value of the first output from the ticket transaction.
+  Due to the way tickets are constructed, we know that this is always the 'price'
+  of any given ticket.
```

Comment:
This is an implementation detail that we don't need to discuss here.  We should just document that it is the ticket price.  If a ticket was purchased with a price greater than required, reporting that value as the ticket price should be considered a bug and fixed.